PIL/Image.py

```diff
@@ -1589,11 +1591,10 @@ def putpixel(self, xy, value):
         :param value: The pixel value.
         """
 
-        self.load()
         if self.readonly:
             self._copy()
             self.pyaccess = None
```

Comment:
`self._copy()` already sets `pyaccess` to None

Reply:
I have created a separate PR for this, #2714